sdk/turing/router/config/autoscaling_policy.py

```diff
@@ -68,7 +68,10 @@ def target(self) -> str:
 
     @target.setter
     def target(self, target: str):
-        assert target.isnumeric()
+        try:
+            float(target)
+        except ValueError:
+            raise AssertionError("Target value must be a number")
```

Comment:
This fix should've been made some time ago because we have been allowing decimal (float) values for autoscaling targets to be specified. `isnumeric` returns false if a decimal value is passed.